content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -276,49 +378,55 @@ The following manifest defines an Ingress that sends traffic to your Service via
    Version: 1.0.0
    Hostname: web-55b8c6998d-8k564
    ```
+<!--
+1. Optionally, you can also visit `hello-world.example` from your browser.
+-->
+5. 可选步骤：你也可以从你的浏览器中访问 `hello-world.example`。
 
+   {{< note >}}
```

Comment:
`{{< /note >}}` is missing, which causes the ci failed.

Reply:
@windsonsea thank,fix it.